src/Microsoft.AspNet.Mvc.TagHelpers/JavaScriptHelpers.cs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using System.Text;
+
+namespace Microsoft.AspNet.Mvc.TagHelpers
```

Comment:
We *could* make it public, just we expect one of those methods to disappear when `WebUtility` is updated.